src/modules/networkcfg/main.py

```diff
@@ -53,4 +53,13 @@ def run():
             except FileExistsError:
                 pass
 
+    # We need to overwrite the default resolv.conf in the chroot.
+    source_resolv = "/etc/resolv.conf"
+    target_resolv = os.path.join(root_mount_point, "etc/resolv.conf")
+    if os.path.exists(source_resolv):
```

Comment:
I would like this to happen only if `target_resolv` doesn't exist already, so we don't go overwriting something that a distro might want to ship as a default in the rootfs.